clang/lib/Sema/SemaDeclAttr.cpp

```diff
@@ -3175,6 +3152,26 @@ static void handleWorkGroupSize(Sema &S, Decl *D, const ParsedAttr &AL) {
       return;
   }
 
+  Expr *XDimExpr = AL.getArgAsExpr(0);
+
+  // If no attribute argument is specified, set the second and third argument
+  // to the default value 1, but only if the sycl:: or intel::
+  // reqd_work_group_size spelling was used.
```

Comment:
@schittir,
Intel:: spelling of reqd_work_group_size attribute has been removed here: https://github.com/intel/llvm/pull/4433.

You do not need to check the intel:: spelling anymore. Please rebase your PR.

Reply:
Thank  you.